components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -1751,6 +1751,20 @@ def seppuku(self, softclose=False):  # pragma: no cover
         session is really terminate disregarding its connection refcount.
 
         :param softclose:   Boolean
+
+        ** Deprecated ** Use :meth:`terminateAllClients`.
+        Our apologies for any offense caused by this previous method name.
+        """
+        warnings.warn("Deprecated. Use terminateAllClients",
+                      DeprecationWarning)
+        self.terminateAllClients(softclose)
+
+    def terminateAllClients(self, softclose=False):  # pragma: no cover
+        """
+        Terminates connection with killSession(). If softclose is False, the
+        session is really terminate disregarding its connection refcount.
```

Comment:
@joshmoore, thoughts? (I can certainly rename, but as for a clearer docstring...)